config.go

```diff
@@ -75,6 +76,12 @@ type GRPCConfig struct {
 	WS int32 `json:"ws"`
 }
 
+type UDPConfig struct {
+	Port              int           `json:"port"`
+	Host              string        `json:"host"`
+	Protos            []string      `json:"protos"`
```

Comment:
Makefile target to compile protos are missing ?

Reply:
I'd love to add `make proto` in the Makefile. Please find the .pb.go files under /schema for now.

Reply:
yes,